articles/00-DownloadingTheXRTK.md

```diff
@@ -58,7 +58,7 @@ You can either:
 * Clone the development XRTK repository into a folder and start building your project from there, pulling down the configured submodules in that project.
 * Clone the UPM branch in to a live project as a Submodule in your source, adding the additional submodules as required
 
-For more information, you can follow [this guide here]()
+For more information, you can follow [this guide here](https://www.rageagainstthepixel.com/expert-import-mrtk/)
```

Comment:
Not all the steps on that are required. Just adding the submodule project.

The linking is all handled in the editor now.

Reply:
Ok, will just update the instructions and remove the reference